src/UI/Art/SpriteArtist/SpriteInfoView.java

```diff
@@ -0,0 +1,141 @@
+
+  public SpriteInfoView(InfoController control)
+  {
+    Dimension dims = Driver.getInstance().gameGraphics.getScreenDimensions();
+    mapViewWidth = dims.width;
+    mapViewHeight = dims.height;
+    myControl = control;
+  }
+
+  @Override
+  public Dimension getPreferredDimensions()
+  {
+    return new Dimension(mapViewWidth, mapViewHeight);
+  }
+
+  @Override
+  public void setPreferredDimensions(int width, int height)
+  {
+    // The user wants to use a specific amount of screen. Figure out how many tiles to draw for them.
+    mapViewWidth = width;
+    mapViewHeight = height;
+    // Let SpriteOptions know we are changing things.
+    SpriteOptions.setScreenDimensions(mapViewWidth, mapViewHeight);
+
+    dimensionsChanged = true; // Let render() know that the window was resized.
+  }
+
+  @Override
+  public void render(Graphics g)
+  {
+    DiagonalBlindsBG.draw(g);
+
+    // Get the CO info menu.
+    int drawScale = SpriteOptions.getDrawScale();
+    
+    int paneOuterBuffer = 4*drawScale; // sets both outer border and frame border size
+    int paneHSize = (int) (mapViewWidth*0.7) - paneOuterBuffer*4; // width of the BG color area
+    int paneVSize = (int) (mapViewHeight   ) - paneOuterBuffer*4; // height ""
+
+    // Get the current menu selections.
+    Commander co = myControl.getSelectedCO();
+    InfoPage page = myControl.getSelectedPage();
+
+    // Draw the commander art.
+    BufferedImage COPic = SpriteLibrary.getCommanderSprites(myControl.getSelectedCOInfo().name).body;
+    // justify bottom/right
+    g.drawImage(COPic, mapViewWidth - COPic.getWidth()*drawScale, mapViewHeight - COPic.getHeight()*drawScale,
+        COPic.getWidth()*drawScale, COPic.getHeight()*drawScale, null);
+
+    if (null != co)
+      CommanderOverlayArtist.drawCommanderOverlay(g, co, false);
+    
+    // add the actual info
+    g.setColor(SpriteUIUtils.MENUFRAMECOLOR);
+    g.fillRect(  paneOuterBuffer,   paneOuterBuffer, paneHSize + 2*paneOuterBuffer, paneVSize + 2*paneOuterBuffer);
+    g.setColor(SpriteUIUtils.MENUBGCOLOR);
+    g.fillRect(2*paneOuterBuffer, 2*paneOuterBuffer, paneHSize                    , paneVSize                    );
+    
+    // TODO: consider drawing all pages as one big image, so the user can scroll smoothly through it regardless of screen size
+    switch (page.pageType)
+    {
+      case CO_HEADERS:
+        int overlayHeight = 30*drawScale;
+        int heightOffset = 0;
+        for (Commander CO : myControl.getGame().commanders)
+        {
+          BufferedImage overlayPic = SpriteLibrary.createTransparentSprite(100*drawScale, overlayHeight);
+          CommanderOverlayArtist.drawCommanderOverlay(overlayPic.getGraphics(), CO, true);
+          g.drawImage(overlayPic,2*paneOuterBuffer, 2*paneOuterBuffer + heightOffset, null);
+          heightOffset += overlayHeight;
+        }
+        break;
```

Comment:
I'd like to add income and unit count here, but I don't want to duplicate code and I'm not sure how best to split up the parsing that happens down below. Any ideas?

It's probably also worth caching the results of all these calculations so they aren't done every frame, but I'm not super worried about that at this point.